interp/handler_test.go

```diff
@@ -38,6 +38,18 @@ func blacklistNondevOpen(ctx context.Context, path string, flags int, mode os.Fi
 	return testOpenHandler(ctx, path, flags, mode)
 }
 
+var runnerCtx = struct{}{}
```

Comment:
use `&struct{}{}`, so that you can just do `runnerCtx` later instead of `&runnerCtx`

Reply:
Also add a short comment, like:

```
// runnerCtx allows us to give handler functions access to the Runner, if needed.
```

Reply:
re: `&struct{}{}` — does it matter that that might not be unique?  See [here](https://dave.cheney.net/2014/03/25/the-empty-struct):

> Interestingly, the address of two struct{} values may be the same.
> 
> ```go
> var a, b struct{}
> fmt.Println(&a == &b) // true
> ```

Reply:
Ah, right. @cclerget please use a non-empty allocated value, like `= new(int)`

Reply:
Fixed